webrtc.html

```diff
@@ -5236,6 +5240,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <var>sender</var>.</p>
                 </li>
                 <li>
+                  <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                  <code>true</code>, abort these steps.</p>
+                </li>
+                <li>
```

Comment:
I think we want to return a rejected promise here since it's an API call, not just abort.

And in the replaceTrack step 9.3, we need the queued task to do a closed state check before resolving the promise.